a10_octavia/controller/worker/tasks/a10_compute_tasks.py

```diff
@@ -39,12 +39,16 @@ def execute(self, amphora_id, loadbalancer=None,
                 server_group_id=None, ports=None, network_list=None):
 
         ports = ports or []
-        network_ids = CONF.a10_controller_worker.amp_boot_network_list[:]
+        mgmt_net = CONF.a10_controller_worker.amp_mgmt_network
+        network_ids = [mgmt_net] if mgmt_net else []
+        network_ids.extend(CONF.a10_controller_worker.amp_boot_network_list[:])
```

Comment:
Do we need to consider the case that:
- amp_mgmt_network is already in amp_boot_network_list or,
- amp_license_network is in network_ids (but not the first one)